exercises/typescript/08-classes/02-problem/src/classes/specialist.ts

```diff
@@ -0,0 +1,4 @@
+import DinoKeeper from "../../../01-solution/src/classes/hello-dino";
```

Comment:
The `problem` and `solution` folders shouldn’t reference each other (they should be completely independent).